setup.py

```diff
@@ -7,7 +7,15 @@
 DESCRIPTION = "A set of python modules for multiview learning"
```

Comment:
remove this file from PR since setup.py is its own PR and should have been merged in already. I think if you rebase from master this file will probably just go away.

Reply:
There was an issue with the setup.py file in master. I'll remove this from the commit but please review the setup.py PR

Reply:
I think this is done right? going to go ahead and resolve